src/commands/run-build.yml

```diff
@@ -1,13 +1,16 @@
 description: >
-  Run builds using the MATLAB build tool. Use this command to run the tasks in the plan returned
-  by a file named buildfile.m in the root of your repository. To use the run-build command, you
-  need MATLAB R2022b or a later release.
+  Run a build using the MATLAB build tool. Use this command to run the tasks specified in a file
+  named `buildfile.m` in the root of your repository. To use the `run-build` command, you need MATLAB
+  R2022b or a later release. MATLAB exits with exit code 0 if the tasks run without error. Otherwise,
+  MATLAB terminates with a nonzero exit code, which causes the command to fail. The command uses the
```

Comment:
I would remove this statement from the description here and add it to the tasks parameter description if possible:

"MATLAB exits with exit code 0 if the tasks run without error. Otherwise,
  MATLAB terminates with a nonzero exit code, which causes the command to fail."